README.md

```diff
@@ -12,9 +12,9 @@ Currently, this setup will only work in AWS US East (N.Virginia) and West (Orego
 
 ## How to deploy ACS Cluster on AWS
 ### Prerequisites
-* You need a hosted zone e.g. example.com. See [Creating a Public Hosted Zone](https://docs.aws.amazon.com/Route53/latest/DeveloperGuide/CreatingHostedZone.html)
-* An SSL certificate for the Elastic Load Balancer and the domains in the hosted zone [Creating SSL Cert](https://docs.aws.amazon.com/elasticloadbalancing/latest/classic/ssl-server-cert.html)
-* For some of the images from the Helm installation access to [Alfresco's repository in quay.io](https://quay.io/repository/alfresco) is required.
+* You need a hosted zone e.g. example.com. See the AWS documentation on [Creating a Public Hosted Zone](https://docs.aws.amazon.com/Route53/latest/DeveloperGuide/CreatingHostedZone.html).
+* An SSL certificate for the Elastic Load Balancer and the domains in the hosted zone. See the AWS documentation on [Creating SSL Certificates](https://docs.aws.amazon.com/elasticloadbalancing/latest/classic/ssl-server-cert.html).
+* Protected Docker images from Quay.io are used during the Helm deployment. You need access to a secret with credentials to be able to pull those images. Alfresco customers can request their credentials by logging a ticket at https://support.alfresco.com.
```

Comment:
I think a better word as Protected would be Private

Reply:
Updated to 'Private'